nav2_dwb_controller/nav_2d_utils/include/nav_2d_utils/odom_subscriber.hpp

```diff
@@ -66,6 +67,9 @@ class OdomSubscriber
       nh.create_subscription<nav_msgs::msg::Odometry>(odom_topic,
         [&](const nav_msgs::msg::Odometry::SharedPtr msg) {odomCallback(msg);},
         1);
+    nh.get_parameter_or("min_x_velocity_threshold", min_x_velocity_threshold_, 0.0001);
```

Comment:
I'd add in `min_x_velocity_threshold` and `min_theta_velocity_threshold` above in `nav2_params.yaml` so that it fully describes all the parameters available.